integration_tests/loop_10.py

```diff
@@ -10,42 +10,91 @@ def with_break_for():
 def with_break_while():
     i: i32 = 0
     while i < 4:
-        print(i)
+        i += 1
         break
     else:
-        print(10)
         assert False
 
 
 def no_break_for():
     i: i32
+    j: i32 = 0
     for i in range(2):
-        print(i)
+        j += 1
     else:
-        print(10)
+        assert j == 2
         return
     assert False
 
 def break_in_if_for():
     i: i32
+    j: i32 = 0
     for i in range(2):
-        print(i)
+        j += 1
         if i == 1:
             break
     else:
-        print(10)
         assert False
+    assert j == 2
 
 def nested_loop_for_for():
     i: i32
     j: i32
     for i in range(2):
-        print("outer: " + str(i))
         for j in range(10, 20):
-            print("  inner: " + str(j))
             break
     else:
-        print("no break in outer loop")
+        return
+    assert False
```

Comment:
No need for return here. Just increment some variable (inside the `loops`, before the `breaks`, etc.). You can use different variables for different loops here. 
At the end of the function print the final values for the variables and add asserts for them. I think this would make it a much more robust test. You can follow this approach for other tests as well. Thanks!